openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +89,123 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+
+== Post installation steps
+
+
+=== Install jump server
+
+To be able to SSH to any of the OCP cluster machines a _jump server_ is required.
+
+==== Requirements
+
+1. Obtain the name of the network interface created by the OCP installation process
```

Comment:
"by the OCP installation process". Is it an ansible process or bash script or ... ? You should be more precise here and ideally point to the documentation about we install ocp

Reply:
This document describes the OCP installation process in it's whole. The installation process is a previous point of the document. I'll try improving the description anyway.